worker/work.go

```diff
@@ -103,7 +103,17 @@ func sendSpans(tx *apm.Transaction, n int) {
 	go func() {
 		defer close(done)
 		for i := 0; i < n; i++ {
-			tx.StartSpan("I'm a span", "gen.era.ted", nil).End()
+			span := tx.StartSpan("I'm a span", "gen.era.ted", nil)
+			resource := "service-1"
+			if n % 2 == 0 {
+				resource = "service-2"
+			}
+			span.Context.SetDestinationService(apm.DestinationServiceSpanContext{
+				Name: resource,
+				Resource: resource,
+			})
+			time.Sleep(time.Duration(rand.Intn(10000000)))
```

Comment:
```suggestion
			span.Duration = time.Duration(rand.Intn(int(10 * time.Millisecond)))
```

Unless there's another reason to sleep, I'd suggest explicitly setting the span duration instead of sleeping. I presume you're just wanting to randomise the span duration to check that we aggregate the durations correctly? Sleeping will change the load generation as well.